database/mssql/scripts/dbscripts/PSP_PIMS_S10_00/Build/83_DML_LEASE_LOAD.sql

```diff
@@ -0,0 +1,318 @@
+/* -----------------------------------------------------------------------------
+Insert leases for testing
+. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 
+Author        Date         Comment
+------------  -----------  -----------------------------------------------------
+Doug Filteau  2021-Aug-19  Initial version
+----------------------------------------------------------------------------- */
+
+-- Insert the lease data
+INSERT INTO [dbo].[PIMS_LEASE]([LEASE_PURPOSE_TYPE_CODE], [LEASE_PURPOSE_SUBTYPE_CODE], [LEASE_STATUS_TYPE_CODE], [LEASE_PMT_FREQ_TYPE_CODE], [LEASE_PROGRAM_TYPE_CODE], [TENANT_ID], [TFA_FILE_NO], [PS_FILE_NO], [START_DATE], [RENEWAL_DATE], [EXPIRY_DATE], [LEASE_AMOUNT], [INSURANCE_START_DATE], [INSURANCE_END_DATE], [SECURITY_START_DATE], [SECURITY_END_DATE], [INSPECTION_DATE], [INSPECTION_NOTES], [LEASE_NOTES], [UNIT], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_DIRECTORY], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_DIRECTORY])
```

Comment:
The following non-nullable columns are causing all of these inserts to fail: [expired], [HAS_PHYSICAL_FILE], [HAS_DIGITAL_FILE], [HAS_PHYSICAL_LICENSE], [HAS_DIGITAL_LICENSE]

Reply:
Got ahead of the build.  I'll amend the script and the data to the correct value for your build.

Reply:
There are no defaults in my version of the database.